### DISCUSSION TITLE
Idea for new mods

### DISCUSSION THREAD

**Discussion by stani63** [ramensoftware/windhawk-mods#394] [General]
Idea for new mods in W11.

**Deactivating the lock screen** (CMD) `powercfg /SETDCVALUEINDEX SCHEME_CURRENT SUB_NONE CONSOLELOCK 0`  work fine in Home (in Pro I don't know).

**Disable automatic user logout after disk sleep/shutdown** (CMD) `powercfg /SETACVALUEINDEX SCHEME_CURRENT SUB_NONE CONSOLELOCK 0`

**Comment by anishsane** [comment]
Lock screen clock - show seconds.
(Not sure if possible, because lock-screen is a secure screen)
